pkg/api/apiloader.go

```diff
@@ -35,6 +35,44 @@ func (a *Apiloader) LoadContainerServiceFromFile(jsonFile string, validate, isUp
 	return a.DeserializeContainerService(contents, validate, isUpdate, existingContainerService)
 }
 
+// LoadDefaultContainerServiceProperties loads the default API model
+func LoadDefaultContainerServiceProperties() (TypeMeta, *vlabs.Properties) {
+	return TypeMeta{APIVersion: vlabs.APIVersion}, &vlabs.Properties{
+		OrchestratorProfile: &vlabs.OrchestratorProfile{
+			OrchestratorType:    "kubernetes",
+			OrchestratorVersion: "1.10.3",
```

Comment:
Why 1.10.3? I would vote for OrchestratorRelease 1.10 that way it picks the latest patch of 1.10